history/characters/340000_night_elf.txt

```diff
@@ -1,16 +1,52 @@
-#Dynasty 128000
+#Dynasty 128000 # Mar'alith
 340000={
-	name=Mar'alith
+	name=Pas'draen
 	dynasty=128000
-	dna="kinkbpfiibj"
+	dna="kpekbpgnfeg"
 	culture=night_elf religion=kaldorei_religion
 	martial=5 diplomacy=6 stewardship=7 intrigue=3 learning=8
 	add_trait=skilled_tactician add_trait=diligent add_trait=brave add_trait=patient add_trait=strategist
 	disallow_random_traits = yes
```

Comment:
If we don't have enough lore info, I would remove it to make other (traits you added above won't be touched) character traits random.